src/main/kotlin/org/rust/ide/injected/RsDoctestLanguageInjector.kt

```diff
@@ -129,6 +129,7 @@ private fun findDoctestInjectableRanges(text: String, elementType: IElementType)
     if (tripleBacktickIndices.size < 2) return emptySequence() // no code blocks in the comment
 
     val infix = RsDocKind.of(elementType).infix
+    val isBlockInfixUsed = text.count { it == '\n' } == "\\n[\\s]*\\*".toRegex().findAll(text).count()
```

Comment:
Looks like rustdoc uses other rules to check it. I don't sure but looks like it checks only first line of the comment body. E.g. this is comment is infixed:
![image](https://user-images.githubusercontent.com/3221931/58433206-bd533800-80be-11e9-8ae3-a552f6ee6bda.png)

Reply:
Are you sure? On my machine this comment 
``` rust
/*! 
* description header
* bar
foo
* bar
*/
```
results in following documentation
![image](https://user-images.githubusercontent.com/8453142/58434100-b3cbcf00-80c2-11e9-806c-08e60a68d147.png)

and your example is broken as well
![image](https://user-images.githubusercontent.com/8453142/58434442-56388200-80c4-11e9-8c38-e8a2cf646525.png)


which doesn't looks like rustdoc tried to remove infix

Reply:
Hmm, really, you're right